backend.native/tests/interop/j2objc/ObjcTest.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
```

Comment:
// TODO replace this with code generated from J2ObjC when done.

Reply:
Done.